keyboards/meletrix/zoom_pad/keymaps/southpaw/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
```

Comment:
Enabling here via makes this keymap somewhat pointless. A user would just flash the `via` keymap and configure the layout options within its UI.